tests/end-to-end/upgrade/pkg/tests/eventmesh-migration/helpers/event.go

```diff
@@ -0,0 +1,261 @@
+package helpers
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"k8s.io/apimachinery/pkg/labels"
+
+	apiv1 "github.com/kyma-project/kyma/components/event-bus/apis/eventing/v1alpha1"
+
+	subApis "github.com/kyma-project/kyma/components/event-bus/apis/eventing/v1alpha1"
+	ebClientSet "github.com/kyma-project/kyma/components/event-bus/client/generated/clientset/internalclientset"
+
+	"github.com/avast/retry-go"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"knative.dev/eventing/pkg/apis/eventing/v1alpha1"
+	eventingclientv1alpha1 "knative.dev/eventing/pkg/client/clientset/versioned/typed/eventing/v1alpha1"
+	"knative.dev/pkg/apis"
+	duckv1 "knative.dev/pkg/apis/duck/v1"
+)
+
+const timeout = time.Second * 30
+
+func SendEvent(target, eventType, eventTypeVersion string) error {
```

Comment:
"SendEventAndVerifyResponse" would be more accurate.

Reply:
What is the point of just sending an event without verifying? SendEvent implicitly means an event is sent successfully or it failed due to so and so reason. `SendEventAndVerifyResponse` is nice to have but it would add verbosity without real value IMO.